### PR TITLE
Swift modules use mangled name which is different than the klassName so get the name from the metadata

### PR DESCRIPTION
**Pull request by KristinaKoeva**

### CODE REVIEW

File: src/NativeScript/TypeFactory.mm [hunk 1]
```diff
@@ -271,11 +271,10 @@
     Structure* prototypeStructure = ObjCPrototype::createStructure(vm, globalObject, parentPrototype);
     ObjCPrototype* prototype = ObjCPrototype::create(vm, globalObject, prototypeStructure, metadata);
 
-    CString klassNameCString = klassName.utf8();
-    Class klass = objc_getClass(klassNameCString.data());
+    Class klass = objc_getClass(metadata->name());
     if (!klass) {
         SymbolLoader::instance().ensureFramework(metadata->topLevelModule()->getName());
-        klass = objc_getClass(klassNameCString.data());
+        klass = objc_getClass(klassName.utf8().data());
```
Comment (jasssonpet) [inline src/NativeScript/TypeFactory.mm]: I was just wondering if this should be `metadata->name()` also?

Comment (ivanbuhov) [inline src/NativeScript/TypeFactory.mm]: Yes, it should `metadata->name()`.

Comment (KristinaKoeva) [inline src/NativeScript/TypeFactory.mm]: will change it :disappointed:

**Comment by jasssonpet** [comment]
:+1:

**Pull request merged**